Player.h

```diff
@@ -0,0 +1,27 @@
+#ifndef PLAYER_H
+#define PLAYER_H
+
+#include <array>
+#include "Card.h"
+#include "Deck.h"
+#include "g_constants.h"
+
+using PlayerHand = std::array<Card, g_constants::g_maximumHand>;
+
+class Player
+{
+private:
+    int m_score{};
+    PlayerHand m_hand{};
```

Comment:
This isn't being used at all.

I think it's good to keep track of the current hand, but an array is probably not ideal, since it's fixed size. Use a vector and reserve a capacity of 7 in the constructor to avoid unnecessary copies.

With a vector you can then see the contents, check the size of the hand etc.